task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+	if(num%3!=0 && num%5!=0) return num
+	else {
```

Comment:
you don't need else here becuase in above statement you have return in such cases else can be omitted

E.g.
```
if ( condtion ) {
    return value1;
}
return value2;
```